lib/httplib/httplib_test.go

```diff
@@ -374,6 +375,67 @@ func TestSetIndexContentSecurityPolicy(t *testing.T) {
 			}
 		})
 	}
+
+	indexCSPStringCache = newCSPCache()
```

Comment:
Why is this necessary?

Reply:
Same question - if it's a cleanup, then let's make sure it's done as such.